src/app/[locale]/(main)/explore/page.tsx

```diff
@@ -1,3 +1,212 @@
+'use client'
+
+import React, { useState, useMemo, useEffect } from 'react'
+import FilterSelect, {
+  domainLabelOptions,
+  goalLabelOptions,
+  languageLabelOptions,
+  technicalLabelOptions,
+} from '@/components/ui/FilterSelect'
+import ProjectCard from '@/components/ui/ProjectCard'
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select'
+import {
+  AlertCircle,
+  Briefcase,
+  Code,
+  Goal,
+  Languages,
+  Search,
+} from 'lucide-react'
+import { Input } from '@/components/ui/input'
+import { Label } from '@/components/ui/label'
+import MyNavbar from '@/components/ui/MyNavbar'
+import BackgroundImage from '@/components/ui/BackgroundImage'
+import useFetchInfiniteProjects from '@/components/features/projects/hooks/useFetchInfiniteProjects'
+import { useFilters } from '@/components/features/filters/FilterContext'
+import { useInView } from 'react-intersection-observer'
+import InfiniteScrollSpin from '@/components/ui/InfiniteScrollSpin'
+
 export default function ExplorePage() {
-  return <div>ExplorePage</div>;
+  const [sortBy, setSortBy] = useState<'recent' | 'popular'>('recent')
+  const {
+    uiFilters,
+    setUiFilters,
+    debouncedFilters,
+    updateTechnical,
+    updateDomain,
+    resetTechnical,
+    resetDomain,
+    resetGoal,
+    resetLanguage,
+  } = useFilters()
+  const { ref, inView } = useInView()
+
+  const {
+    data,
+    fetchNextPage,
+    hasNextPage,
+    isFetchingNextPage,
+    isLoading,
+    error,
+  } = useFetchInfiniteProjects({
+    search: debouncedFilters.search,
+    technical: debouncedFilters.technical,
+    domain: debouncedFilters.domain,
+    goal: debouncedFilters.goal,
+    languageName: debouncedFilters.languageName,
+  })
+
+  useEffect(() => {
+    if (inView && hasNextPage) {
+      fetchNextPage()
+    }
+  }, [inView, hasNextPage, fetchNextPage])
+
+  const sortedProjects = useMemo(() => {
+    const allProjects = data?.pages.flatMap((page) => page.projects) || []
+    return [...allProjects].sort((a, b) => {
+      if (sortBy === 'recent') {
+        return (
+          new Date(b.createdAt || 0).getTime() -
+          new Date(a.createdAt || 0).getTime()
+        )
+      }
+      return b.id - a.id
+    })
+  }, [data, sortBy])
+
+  return (
+    <div className="flex w-full relative items-center justify-center ">
+      <BackgroundImage bgImage="/images/bg-explore.png" height={''}>
```

Comment:
@vadimkim0203 Regarding the BackgroundImage component:
For truly dynamic backgrounds, we’d need inline styles or Next.js <Image>. 
However, for most static pages, a simple Tailwind <div> is cleaner and safer.

I’d recommend removing the BackgroundImage component and using:

```suggestion
<div className="fixed inset-0 bg-[url('/images/bg-explore.png')] bg-cover bg-center bg-no-repeat -z-10" />
```

FYI: @kimdahee0815